app/views/games/show.html.erb

```diff
@@ -18,3 +22,18 @@
 </div>
 
 <div class='clear'></div>
+
+<script>
+  $(function() {
+    $( ".draggable" ).draggable();
+    
+    $("td").droppable({
+      drop: function( event, ui ) {
+        $(ui.draggable).detach().css({top: 0,left: 0}).appendTo(this);

```

Comment:
I know you know this but eventually you'll have to move the piece back if the move is invalid, just keep this in mind.  You may want to use $.ajax to call the back end on the drop then only append the element on success...